training/src/main/scala/com/airbnb/aerosolve/training/AdditiveModelTrainer.scala

```diff
@@ -80,9 +78,9 @@ object AdditiveModelTrainer {
     * During each iteration, we:
     *
     * 1. Sample dataset with subsample (this is analogous to mini-batch sgd?)
-    * 1. Repartition to numBags (this is analogous to ensemble averaging?)
-    * 1. For each bag we run SGD (observation-wise gradient updates)
-    * 1. We then average fitted weights for each feature and return them as updated model
+    * 2. Repartition to numBags (this is analogous to ensemble averaging?)
+    * 3. For each bag we run SGD (observation-wise gradient updates)
+    * 4. We then average fitted weights for each feature and return them as updated model
     *
```

Comment:
Scaladoc requires numbered list use exact same number and it will increment them automatically. Totally weird! https://gist.github.com/VladUreche/8396624#file-gistfile1-txt-L127